src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -524,6 +524,22 @@ std::tuple<std::optional<int>, std::optional<int>> DetermineNavalBase(const std:
 }
 
 
+int DetermineAirbaseLevel(int total_factories, int total_infrastructure)
+{
+   constexpr int kNumFactoriesPerAirbase = 4;
+   constexpr int kAirbasesForInfrastructureLevel = 3;
+
+   int airbase_level = 1;
```

Comment:
with this new method, do we want default airbase level to be 0? I think every single state having an airbase is kinda weird